.github/workflows/test_template.yaml

```diff
@@ -1,20 +1,30 @@
-name: Test
-on: [push, pull_request]
+name: Workflow Test Template
+on: 
+  workflow_call:
+    inputs:
+      runs-on:
+        required: false
+        type: string
+        default: ubuntu-latest
+      targets:
+        required: false
+        type: string
+        default: "['linux-unit-test-1-cpu', 'linux-unit-test-4-cpu']"
```

Comment:
These default values don't match any of the targets available. I suggest either setting the default as empty (`"[]"`) or to one target that actually exists.